docs/asgi-frameworks.md

```diff
@@ -11,14 +11,15 @@ We can think about the ASGI framework support without referencing an existing im
 Let's invent an API for a non-existent microframework to demonstrate things further. This could represent *any* ASGI framework application:
 
 ```python
+import mangum.adapter
 import framework
-from mangum import Mangum
+from mangum import Mangum, Request
 
 app = framework.applications.Application()
 
 
 @app.route("/")
-def endpoint(request: framework.requests.Request) -> dict:
+def endpoint(request: Request) -> dict:
```

Comment:
This would be unchanged because the request in the framework is different than the request used in Mangum itself, though I can change this later after merging.